### PR TITLE
chore(ci): remove tokens and add windows to ci

### PR DESCRIPTION
**Pull request by aajtodd**
## Issue \#

N/A

## Description of changes

* Add windows target to CI
* Removes unnecessary tokens that were required to checkout dependencies when repos were private

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -5,9 +5,6 @@ on:
     branches: [ main ]
   pull_request:
     branches: [ main ]
-    paths-ignore:
```
Comment (kggilmer) [inline .github/workflows/continuous-integration.yml:8]: question

why did you remove this?

**Review by kggilmer — APPROVED** [review]
Nice work!  I was going to need windows coverage as part of my current task so perfect timing.

File: .github/workflows/continuous-integration.yml (hunk 1 above)
Comment (aajtodd) [reply .github/workflows/continuous-integration.yml:8]: Because as we discovered in a different branch if we actually trigger this then CI won't run (as you would expect). The issue is all of our checks for this workflow are marked as required status checks. You can't merge the branch without the status check passing which will never be generated because the workflow never runs.

**Review by ianbotsf — APPROVED** [review]
